### PR TITLE
File per connection

### PR DESCRIPTION
**Pull request by simonl2002**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -136,29 +154,60 @@ function run(keys) {
...
-                        2: city,
-                        time: Date.now()
-                        }) + '\n'
-                    );
-                }
+                locationData = {
+                    0: 'location',
+                    1: null,
+                    2: city,
+                    time: Date.now()
```
Comment (johnryan) [inline app.js:180]: @simonl2002 why was this changed?

File: app.js [hunk 2]
```diff
@@ -136,29 +154,60 @@ function run(keys) {
...
+                };
             });
         }
 
         console.log('connected', ua, referer, clientid);
 
         client.on('message', msg => {
             const data = JSON.parse(msg);
+
+            if (!tempStream || (data[0] === 'create' && data[1] !== peerConnectionId)) {
```
Comment (johnryan) [inline app.js:190]: just a suggestion (though it might require more changes in the `switch` below) is that it might be a little cleaner to write `const [event, connId] = data`

then the code would read a little cleaner i.e. 
```
if (!tempStream || event === 'create' && connId !== peerConnectionId)) {
```
and other references to randomish data indexing might be unnecessary

File: app.js (hunk 1 above)
Comment (simonl2002) [reply app.js:180]: So `tempStream` is no longer assigned at this point and isn't created until we start receiving messages. Additionally we need to stick this location information into all the files created. So I just populate the variable and reuse it to write into the streams when they get created.

Comment (johnryan) [reply app.js:180]: gotcha

**Review by johnryan — APPROVED** [review]
lgtm
